### PR TITLE
style: update some styles

### PR DESCRIPTION
**Pull request by guoqqqi**
1. Optimising markdown rendering；
2. Update the mobile home page style；
3. Navigation bar Request demo Change to Dashboard entry；
4. Uniform hover style for hyperlink buttons；

TODO: 
Uniform page style
- [x] blog
- [x] usercase

### CODE REVIEW

File: views/Post/Post.tsx [hunk 1]
```diff
@@ -112,14 +113,29 @@ const Page: NextPage<Props, any> = ({ t, content, data = {}, isSimple }) => {
...
               <SBox1>
+                <h1>{data.title}</h1>
                 <SInner>
-                  <h1>{data.title}</h1>
-                  <span>
-                    {t("common:updatedAt")}&nbsp;
-                      {new Date(data.date).toLocaleDateString()}
-                  </span>
+                  <div className="left">
+                    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSao9hvCLqBfgE5WDNqILk4MZJA00YvgvhFpQ&usqp=CAU" alt="" />
```
Comment (juzhiyuan) [inline views/Post/Post.tsx:120]: Could we also support the `avatar` prop? then we could pass avatar for every author.

File: views/Post/Post.tsx [hunk 2]
```diff
@@ -112,14 +113,29 @@ const Page: NextPage<Props, any> = ({ t, content, data = {}, isSimple }) => {
...
+                      </p>
+                      <p>
+                        {t("common:updatedAt")}&nbsp;
+                        {new Date(data.date).toLocaleDateString()}
+                      </p>
+                    </div>
+                  </div>
+                  <div className="right">
+                    <ul>
+                      <li><a href=""><svg><path d="M22.05 7.54a4.47 4.47 0 0 0-3.3-1.46 4.53 4.53 0 0 0-4.53 4.53c0 .35.04.7.08 1.05A12.9 12.9 0 0 1 5 6.89a5.1 5.1 0 0 0-.65 2.26c.03 1.6.83 2.99 2.02 3.79a4.3 4.3 0 0 1-2.02-.57v.08a4.55 4.55 0 0 0 3.63 4.44c-.4.08-.8.13-1.21.16l-.81-.08a4.54 4.54 0 0 0 4.2 3.15 9.56 9.56 0 0 1-5.66 1.94l-1.05-.08c2 1.27 4.38 2.02 6.94 2.02 8.3 0 12.86-6.9 12.84-12.85.02-.24 0-.43 0-.65a8.68 8.68 0 0 0 2.26-2.34c-.82.38-1.7.62-2.6.72a4.37 4.37 0 0 0 1.95-2.51c-.84.53-1.81.9-2.83 1.13z"></path></svg></a></li>
```
Comment (juzhiyuan) [inline views/Post/Post.tsx:133]: Also, we could only support Twitter first, and set its href to `https://twitter.com/intent/tweet?text=${window.location.href}`

File: views/PostList/PostList.tsx [hunk 3]
```diff
@@ -18,28 +19,13 @@ const PostList: NextPage<Props, any> = ({ t, type, list = [] }) => {
...
       <NextSeo title={t(`common:${type}`)} />
-      <div>
-        <div className="cover">
-          <div className="background"></div>
-          <div className="box">
-            <h1>{t(`common:${type}`)}</h1>
-          </div>
-        </div>
+      <div style={{borderTop: "2px solid #F7FAFC"}}>
+        <PostListCard />
```
Comment (juzhiyuan) [inline views/PostList/PostList.tsx:23]: Why not put contents inside this component to here?

File: data/blog.json [hunk 4]
```diff
@@ -2,173 +2,229 @@
   "zh-CN": [
     {
       "title": "使用 Cypress 获取前端测试覆盖率",
-      "cover": "https://static.apiseven.com/logo/cypress.png",
+      "image": "https://static.apiseven.com/logo/cypress.png",
+      "type": "article",
+      "tags": [],
       "date": "2021-03-02",
-      "path": "/zh/blog/get-front-end-test-coverage-with-cypress"
+      "href": "/zh/blog/get-front-end-test-coverage-with-cypress"
```
Comment (juzhiyuan) [inline data/blog.json:9]: no, `href` mean `hyper link`, use `path` here.

File: _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "A first look at Kubernetes Service APIs"
-avatar: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSao9hvCLqBfgE5WDNqILk4MZJA00YvgvhFpQ&usqp=CAU"
+avatar: "https://avatars.githubusercontent.com/u/25628854?s=460&u=775adf495b52dd8424de0f2857f89c87a7a29dbd&v=4"
 author: "Jin Wei"
-href: ""
+href: "https://github.com/starsz"
```
Comment (juzhiyuan) [inline _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md]: no, https://apisix.apache.org/team/

File: _posts/blog/zh-CN/a-first-look-at-kubernetes-service-api.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "初探 Kubernetes Service APIs"
-avatar: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSao9hvCLqBfgE5WDNqILk4MZJA00YvgvhFpQ&usqp=CAU"
+avatar: "https://avatars.githubusercontent.com/u/25628854?s=460&u=775adf495b52dd8424de0f2857f89c87a7a29dbd&v=4"
 author: "金卫"
-href: ""
+href: "https://github.com/starsz"
```
Comment (juzhiyuan) [inline _posts/blog/zh-CN/a-first-look-at-kubernetes-service-api.md]: ,

File: _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md (hunk 5 above)
Comment (guoqqqi) [reply _posts/blog/en-US/a-first-look-at-kubernetes-service-api.md]: Already changed

File: _posts/blog/zh-CN/a-first-look-at-kubernetes-service-api.md (hunk 6 above)
Comment (guoqqqi) [reply _posts/blog/zh-CN/a-first-look-at-kubernetes-service-api.md]: Already changed

File: _posts/blog/en-US/how-Amazon-EKS-and-APISIX-ingress-controller-work-together-to-manage-complex-traffic.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "How Amazon EKS and APISIX ingress controller work together to manage complex traffic"
-avatar: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSao9hvCLqBfgE5WDNqILk4MZJA00YvgvhFpQ&usqp=CAU"
+avatar: ""
```
Comment (juzhiyuan) [inline _posts/blog/en-US/how-Amazon-EKS-and-APISIX-ingress-controller-work-together-to-manage-complex-traffic.md:3]: https://api7-website-1301662268.file.myqcloud.com/202102/rsz_aws-logo-grande.png

How about this one?

Comment (guoqqqi) [reply _posts/blog/en-US/how-Amazon-EKS-and-APISIX-ingress-controller-work-together-to-manage-complex-traffic.md:3]: done

File: _posts/blog/en-US/how-to-build-gateway-based-on-apache-apisix.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "BEKE: How to build a gateway based on Apache APISIX"
-avatar: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSao9hvCLqBfgE5WDNqILk4MZJA00YvgvhFpQ&usqp=CAU"
+avatar: ""
```
Comment (juzhiyuan) [inline _posts/blog/en-US/how-to-build-gateway-based-on-apache-apisix.md:3]: use this as default avatar: https://api7-website-1301662268.file.myqcloud.com/logo/API7-Logo.png

Not in this file, in the avatar component.

**Comment by guoqqqi** [comment]
image
image
Which one is better?

File: _posts/blog/en-US/how-to-build-gateway-based-on-apache-apisix.md (hunk 8 above)
Comment (guoqqqi) [reply _posts/blog/en-US/how-to-build-gateway-based-on-apache-apisix.md:3]: done

**Comment by juzhiyuan** [comment]
image

why not use `object-fit = contain`?

**Comment by juzhiyuan** [comment]
and please fix the failed Vercel ASAP

File: views/Post/Post.tsx [hunk 9]
```diff
@@ -38,7 +40,10 @@ const CodeBlock = ({ language, value }) => {
 };
 
 const Page: NextPage<Props, any> = ({ t, content, data = {}, isSimple }) => {
+  const [shareUrl, setShareUrl] = useState('');
+
   useEffect(() => {
+    setShareUrl(`${window.location.href}`);
```
Comment (juzhiyuan) [inline views/Post/Post.tsx:46]:
```suggestion
    setShareUrl(window.location.href);
```

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**